cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -165,22 +167,22 @@ function reconcile_addons() {
   # Filter out `configured` message to not noisily log.
   # `created`, `pruned` and errors will be logged.
   log INFO "== Reconciling with deprecated label =="
-  ${KUBECTL} ${KUBECTL_OPTS} apply -f ${ADDON_PATH} \
+  ${KUBECTL} "${KUBECTL_OPTS}" apply -f "${ADDON_PATH}" \
     -l ${CLUSTER_SERVICE_LABEL}=true,${ADDON_MANAGER_LABEL}!=EnsureExists \
-    --prune=true ${prune_whitelist_flags} --recursive | grep -v configured
+    --prune=true "${prune_whitelist_flags}" --recursive | grep -v configured
 
   log INFO "== Reconciling with addon-manager label =="
-  ${KUBECTL} ${KUBECTL_OPTS} apply -f ${ADDON_PATH} \
+  ${KUBECTL} "${KUBECTL_OPTS}" apply -f "${ADDON_PATH}" \
     -l ${CLUSTER_SERVICE_LABEL}!=true,${ADDON_MANAGER_LABEL}=Reconcile \
-    --prune=true ${prune_whitelist_flags} --recursive | grep -v configured
+    --prune=true "${prune_whitelist_flags}" --recursive | grep -v configured
 
   log INFO "== Kubernetes addon reconcile completed at $(date -Is) =="
 }
 
 function ensure_addons() {
   # Create objects already exist should fail.
   # Filter out `AlreadyExists` message to not noisily log.
-  ${KUBECTL} ${KUBECTL_OPTS} create -f ${ADDON_PATH} \
+  ${KUBECTL} "${KUBECTL_OPTS}" create -f "${ADDON_PATH}" \
```

Comment:
I suspect KUBECTL_OPTS probably contains multiple flags

Reply:
You are right. Thank you!